lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -761,17 +761,17 @@ function dosomething_campaign_add_shipment_form_vars($node) {
 function dosomething_campaign_is_hot_module(&$vars) {
   $hot_module_enabled = dosomething_helpers_get_variable('node', $vars['nid'], 'hot_module_enabled');
   if ($hot_module_enabled && isset($vars['field_high_season'])) {
-    $today = dosomething_campaign_set_est_timezone(new DateTime());
-    $start_date = dosomething_campaign_set_est_timezone(new DateTime($vars['field_high_season'][0]['value']), TRUE);
-    $end_date = dosomething_campaign_set_est_timezone(new DateTime($vars['field_high_season'][0]['value2']), TRUE, TRUE);
+    date_default_timezone_set('EST');
```

Comment:
you're setting this date default here and in `dosomething_campaign_get_goals_times`